### PR TITLE
[doc] requirements: Add section encouraging DMABUF use for iGPUs

### PR DESCRIPTION
**Pull request by JJRcop**
Also added link in install_libvirt to the new section.

### CODE REVIEW

**Comment by gnif** [comment]
The wording needs to change a bit here. There is no "copy engine" per say under Linux, it's a DMA transfer, as such it offloads the transfer from the CPU, to the GPUs DMA hardware.

> without using an intermediary buffer

It might still use one, it depends on the GPU hardware and driver implementation. iGPUs may even avoid any copy at all as they share system RAM with the GPU and could just use it directly without any copy at all.

> considered necessary for a decent experience.

I would reword this to state it is a requirement.

**Review by gnif — CHANGES_REQUESTED** [review]
.

File: doc/requirements.rst [hunk 1]
```diff
@@ -78,3 +76,22 @@ NVIDIA unlike AMD do not seem to suffer from the same stability issues as AMD
...
+
+While `DMABUF` with the :doc:`KVMFR module <ivshmem_kvmfr>` offers performance
+benefits for all users, for the often bandwidth-starved users with an iGPU on
+their host it's considered a requirement for a decent experience.
+
+When using a normal SHM file, many GPU drivers will copy incoming frames from
+shared memory to an intermediary buffer, then upload it from that buffer to the
+GPU's framebuffer. The KVMFR module will instead use Direct Memory Access (DMA)
+to download incoming frames directly from shared memory, which may depending on
+GPU design eliminate the intermediary buffer. This is especially helpful to iGPU
```
Comment (JJRcop) [inline doc/requirements.rst:93]: > which may depending on
GPU design eliminate the intermediary buffer.

I hope this modification is correct, this is how I understand it.
